hack/jenkins/e2e-runner.sh

```diff
@@ -186,6 +179,37 @@ function dump_cluster_logs() {
     fi
 }
 
+# GCI specific settings.
+# Assumes: JENKINS_GCI_IMAGE_FAMILY
+function setup_gci_vars() {
+  local -r gci_staging_project=container-vm-image-staging
+  local -r image_info="$(gcloud compute images describe-from-family ${JENKINS_GCI_IMAGE_FAMILY} --project=${gci_staging_project} --format=json)"
+  local -r image_description="$(echo ${image_info} | jq '.description')"
+  local -r image_name="$(echo ${image_info} | jq '.name')"
+
+  if [[ "${JENKINS_USE_GCI_VERSION:-}" =~ ^[yY]$ ]]; then
```

Comment:
Because this way it allows both cases? For some reason this has been a convention in k8s' code